app/src/main/java/com/cornellappdev/transit/ui/screens/HomeScreen.kt

```diff
@@ -183,7 +183,7 @@ fun HomeScreen(
                         if (!homeViewModel.searchQuery.value.isBlank() && it.lowercase()
                                 .contains(homeViewModel.searchQuery.value.lowercase())
                         ) {
```

Comment:
We'll probably want to make a generic "fulfills query" function instead of doing this in the future but this is okay for now.